src/app/model/migrations/MigrationFactory.ts

```diff
@@ -1,3 +1,4 @@
+import { MigrationToVersion0002 } from '@app/app/model/migrations/migrations/to0002/MigrationToVersion0002';
```

Comment:
It's a very minor suggestion but could you change the import to this:
```suggestion
import { MigrationToVersion0002 } from './migrations/to0002/MigrationToVersion0002';
```